server/data_common/dataset_metadata.py

```diff
@@ -71,7 +71,16 @@ def get_dataset_metadata_for_explorer_location(dataset_explorer_location: str, a
         dataset_metadata = request_dataset_metadata_from_data_portal(
             data_portal_api_base=app_config.server_config.data_locator__api_base, explorer_url=explorer_url_path
         )
+
         if dataset_metadata:
+            if dataset_metadata['tombstoned']:
```

Comment:
Not sure why the linter doesn't complain, but I think we're enforcing double quotes here.